### PR TITLE
Add "My account" view

### PR DESCRIPTION
**Pull request by abefernan**
This new view accessible by an icon on the header allows the user to connect to Keplr or Ledger and see its address, pubkey, link to the address in an explorer, and balances on the connected chain.

### CODE REVIEW

**Comment by webmaster128** [comment]
I tried loading the preview URL for this PR (https://cosmos-multisig-ui-git-feat-my-account-confio.vercel.app/ ) but unfortunately get a lot of errors. Any idea why?

Bildschirmfoto 2023-12-12 um 09 34 57

**Comment by hashedone** [comment]
This does not to work - I wanted to start the PR, and I started with the preview. All I got is:

> Application error: a client-side exception has occurred (see the browser console for more information).

The Multisig app only flashes for a second, and then I get an error. Something is not working. Browser is FF 118.0.

File: components/dataViews/AccountView/index.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              {explorerLink ? (
+                <Button asChild className="self-center">
+                  <a href={explorerLink} target="_blank">
+                    View in explorer
+                  </a>
+                </Button>
+              ) : null}
+            </div>
+            <div className="mt-4">
+              <h2>PubKey</h2>
```
Comment (webmaster128) [inline components/dataViews/AccountView/index.tsx:54]:
```suggestion
              <h2>Public key</h2>
```

File: components/dataViews/AccountView/BalancesList.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        const newBalances = await client.getAllBalances(walletAddress);
+        setBalances(newBalances);
+      } catch (e: unknown) {
+        setError(e instanceof Error ? e.message : "Failed to get balances");
+        console.error("Get balances error:", e);
+      }
+    })();
+  }, [chain.nodeAddress, setError, walletAddress]);
+
+  return balances.length ? (
```
Comment (hashedone) [inline components/dataViews/AccountView/BalancesList.tsx:34]: How do I get this balance tab? I cannot really verify it on the preview. I assume it requires me to connect kepler, but connecting to kepler (at least on preview) seems not to work.

File: components/dataViews/AccountView/ButtonConnectWallet.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+export default function ButtonConnectWallet({
+  walletType,
+  walletInfoState: [walletInfo, setWalletInfo],
+  setError,
+}: ButtonConnectWalletProps) {
+  const { chain } = useChains();
+  const [loading, setLoading] = useState<LoadingStates>({});
+
+  const connectKeplr = useCallback(async () => {
```
Comment (hashedone) [inline components/dataViews/AccountView/ButtonConnectWallet.tsx:30]: Ok, I see connecting keplr is just added, but it doesn't work for me really.

**Review by hashedone — COMMENTED** [review]
Seems like the explorer account link is working and visible.

I see buttons for Kplr and Ledger connections, but the first one doesn't work (and I can't check the ledger - I don't have the proper setup right now). Not sure if it is only the preview problem or if it doesn't work, and I don't know how to verify it.

Also, I see some balances previewed in the code but cannot see them on preview - I suspect it is not showing up until I connect Kepl/Ledger, which I cannot verify.

My biggest issue with this is that those should be at least three if not four, separate reviews - Ledger/Keplr connection (possibly separated), Explorer Account Link (that could be immediately merged), and balances (or most likely what you mean by "My account").

**Pull request merged**